lms/validation/authentication/_bearer_token.py

```diff
@@ -148,13 +150,13 @@ def _decode_jwt(self, data, **_kwargs):
 
         https://marshmallow.readthedocs.io/en/2.x-line/extending.html#example-enveloping
         """
-        try:
-            jwt = data["authorization"][len("Bearer ") :]
-        except KeyError:
+        if data["authorization"] == marshmallow.missing:
```

Comment:
It used to be that `"authorization"` would simply be missing from `data` and so `KeyError` would be raised. Now `data["authorization"]` is a `marshmallow.missing` value :man_shrugging: